example/cumulus-tf/queue_granules_workflow.asl.json

```diff
@@ -0,0 +1,66 @@
+{
+  "Comment": "Queue Granules",
+  "StartAt": "QueueGranules",
+  "States": {
+    "QueueGranules": {
+      "Parameters": {
+        "cma": {
+          "event.$": "$",
+          "task_config": {
+            "queueUrl": "{$.meta.queueUrl}",
+            "provider": "{$.meta.provider}",
+            "internalBucket": "{$.meta.buckets.internal.name}",
+            "stackName": "{$.meta.stack}",
+            "granuleIngestWorkflow": "${ingest_granule_workflow_name}",
+            "childWorkflowMeta": { "staticValue": "aStaticValue", "interpolatedValueStackName": "{$.meta.stack}" },
```

Comment:
You don't need this line if you aren't testing for it in the spec test, which it doesn't look like you are.